### PR TITLE
[docs] Whats new in 2.1.8

### PR DESCRIPTION
**Pull request by stevebang**

### CODE REVIEW

File: docs/content/latest/architecture/docdb/performance.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The keys stored by DocDB in RocksDB consist of a number of components, where the
 
 The bloom filter needs to be aware of what components of the key need be added to the bloom so that only the relevant SSTable files in the LSM store are looked up during a read operation.
 
-In a traditional KV store, range scans do not make use of bloom filters because exact keys that fall in the range are unknown. However, we have implemented a data-model aware bloom filter, where range scans within keys that share the same hash component can also benefit from Bloom filters. For example, a scan to get all the columns within row or all the elements of a collection can also benefit from bloom filters.
+In a traditional KV store, range scans do not make use of bloom filters because exact keys that fall in the range are unknown. However, you have implemented a data-model aware bloom filter, where range scans within keys that share the same hash component can also benefit from Bloom filters. For example, a scan to get all the columns within row or all the elements of a collection can also benefit from bloom filters.
```
Comment (schoudhury) [inline docs/content/latest/architecture/docdb/performance.md:59]: this change to `you` is incorrect

File: docs/content/latest/benchmark/resilience/jepsen-testing-ysql.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ Yugabyte works diligently to ensure that distributed transactions in YugabyteDB
 
 ### Robust to clock skews
 
-From the Jepsen report: "Whatever the case, this is a good thing for operators: nobody wants to worry about clock safety unless they have to, and YugabyteDB appears to be mostly robust to clock skew. Keep in mind that we cannot robustly test YugabyteDB’s use of `CLOCK_MONOTONIC_RAW` for leader leases, but we suspect skew there is less of an issue than `CLOCK_REALTIME` synchronization."
+From the Jepsen report: "Whatever the case, this is a good thing for operators: nobody wants to worry about clock safety unless they have to, and YugabyteDB appears to be mostly robust to clock skew. Keep in mind that you cannot robustly test YugabyteDB’s use of `CLOCK_MONOTONIC_RAW` for leader leases, but you suspect skew there is less of an issue than `CLOCK_REALTIME` synchronization."
```
Comment (schoudhury) [inline docs/content/latest/benchmark/resilience/jepsen-testing-ysql.md:107]: again this change to `you` is incorrect.

Comment (stevebang) [reply docs/content/latest/benchmark/resilience/jepsen-testing-ysql.md]: fixed here

File: docs/content/latest/architecture/docdb/performance.md (hunk 1 above)
Comment (stevebang) [reply docs/content/latest/architecture/docdb/performance.md]: fixed

File: docs/content/latest/deploy/public-clouds/aws/manual-deployment.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ We now have 2 VMs each in Availability Zones `us-west-2a`, `us-west-2b`, `us-wes
 
 Now that the six nodes have been prepared, the yb-master process will be run on three of these nodes (because RF=3) and yb-tserver will be run on all six nodes. To learn more about YugabyteDB’s process architecture, see [here](../../../../architecture/concepts/universe/).
```
Comment (schoudhury) [inline docs/content/latest/deploy/public-clouds/aws/manual-deployment.md:54]: change `process architecture` to `server architecture`

File: docs/content/latest/deploy/public-clouds/aws/manual-deployment.md [hunk 4]
```diff
@@ -53,7 +53,7 @@ We now have 2 VMs each in Availability Zones `us-west-2a`, `us-west-2b`, `us-wes
...
 
-These install steps are written in a way that we assume that you will run the install steps from another node from which you can access the above six VMs over `ssh`.
+These install steps are written in a way that you assume that you will run the install steps from another node from which you can access the above six VMs over `ssh`.
```
Comment (schoudhury) [inline docs/content/latest/deploy/public-clouds/aws/manual-deployment.md:56]: wrong use of `you`

File: docs/content/latest/releases/whats-new/_index.md [hunk 5]
```diff
@@ -40,38 +40,95 @@ menu:
...
+- Add metrics for RPC queue sizes of services, including YB-Master, YB-TServer, YCQL, and YEDIS. [#4294](https://github.com/yugabyte/yugabyte-db/issues/4294)
+- Add option to edit configuration flags without requiring server restart. [#4433](https://github.com/yugabyte/yugabyte-db/issues/4433)
+- When configuration flags are deleted in the YugabyteDB Admin Console, remove the flags from `server.conf` file. [#4341](https://github.com/yugabyte/yugabyte-db/issues/4341)
+- When creating GCP instances, only use host project when specifying network.
+- When creating a cloud provider configuration, display provider-level (non-k8s) settings for SSH ports and enabling airgapped installations. [#3615](https://github.com/yugabyte/yugabyte-db/issues/3615), [#4243](https://github.com/yugabyte/yugabyte-db/issues/4243), and [#4240](https://github.com/yugabyte/yugabyte-db/issues/4240).
+- After removing a node and then adding a node, check for certificate and key files and create the files if needed. [#4551](https://github.com/yugabyte/yugabyte-db/issues/4551)
+- Update to support Helm 3 deployments. [#4416]((https://github.com/yugabyte/yugabyte-db/issues/4416))
```
Comment (schoudhury) [inline docs/content/latest/releases/whats-new/_index.md:122]: it should also say that Helm 2 is no longer supported and should link to the doc where we show how to migrate an existing Helm 2 universe to Helm 3.

File: docs/content/latest/deploy/public-clouds/aws/manual-deployment.md (hunk 3 above)
Comment (stevebang) [reply docs/content/latest/deploy/public-clouds/aws/manual-deployment.md]: fixed

File: docs/content/latest/deploy/public-clouds/aws/manual-deployment.md (hunk 4 above)
Comment (stevebang) [reply docs/content/latest/deploy/public-clouds/aws/manual-deployment.md]: fixed

**Pull request merged**